pkg/postgres/schema/process_indicators.go

```diff
@@ -45,16 +48,20 @@ const (
 )
 
 // ProcessIndicators holds the Gorm model for Postgres table `process_indicators`.
+// XXX: unique_indicator index is specified manually, as it has to cover
```

Comment:
We already talked about it, but for posterity, we should update the generators to allow for this case.  Though possibly as follow on work.